mvpa2/cmdline/cmd_ofmotionqc.py

```diff
@@ -53,63 +91,85 @@ def run(args):
     from mvpa2.datasets.sources.openfmri import OpenFMRIDataset
```

Comment:
could there be at least a smoke test for this now that we have a sample openfmri 1 subj tutorial_data?